compiler/codegen/OMRCodeGenPhase.cpp

```diff
@@ -505,6 +505,29 @@ OMR::CodeGenPhase::performLowerTreesPhase(TR::CodeGenerator * cg, TR::CodeGenPha
    }
 
 
+void
+OMR::CodeGenPhase::performUncommonCallConstNodesPhase(TR::CodeGenerator * cg, TR::CodeGenPhase * phase)
+   {
+   TR::Compilation* comp = cg->comp();
+   phase->reportPhase(InstructionSelectionPhase);
+
+   if (comp->getOption(TR_TraceCG) || comp->getOption(TR_TraceTrees) || comp->getOptions()->getTraceCGOption(TR_TraceCGPreInstructionSelection))
+      comp->dumpMethodTrees("Pre Uncommon Call Constant Node Trees");
+
+   TR::LexicalMemProfiler mp(phase->getName(), comp->phaseMemProfiler());
+   LexicalTimer pt(phase->getName(), comp->phaseTimer());
+
+   cg->uncommonCallConstNodes();
+
+   if (comp->getOption(TR_TraceCG) || comp->getOption(TR_TraceTrees) || comp->getOptions()->getTraceCGOption(TR_TraceCGPreInstructionSelection))
+      comp->dumpMethodTrees("Post Uncommon Call Constant Node Trees");
+
+   // check interrupt
+   if (comp->compilationShouldBeInterrupted(AFTER_INSTRUCTION_SELECTION_CONTEXT))
```

Comment:
I don't know if you really need to check for interruption after what should be a fairly quick pass, but you'll be reporting the wrong yield point if you're interrupted. Personally I'd get rid of this, but if you want to keep it you may as well report the right thing.